WickedEngine/wiNetwork_Linux.cpp

```diff
@@ -0,0 +1,32 @@
+#include    "wiNetwork.h"
+
+namespace wiNetwork
+{
+    void Initialize() {
```

Comment:
Just calling out that I start the brackets on new lines for functions. If somewhere is not, it's a mistake.

Reply:
Of course. We really need to get a stylesheet made up